experiments/kompanion/pkg/utils/utils.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 Google LLC
```

Comment:
2024

There should be a linter for this. I think I have an issue on it but haven't looked at adding one yet.

Reply:
Fair